tidb-server/main.go

```diff
@@ -207,6 +208,12 @@ func exit() {
 
 func syncLog() {
 	if err := log.Sync(); err != nil {
+		// Don't complain about /dev/stdout as Fsync will return EINVAL.
+		if pathErr, ok := err.(*fs.PathError); ok {
```

Comment:
Just a remind: A better way to decide whether too errors are of the same kind is using `errors.Is()`

https://pkg.go.dev/errors#Is

Because the error may be chained, an error is caused by the another error, and `Is()` function could compare the whole chain.